

fs/cachefs/misc.c: In function `dump_bio':
fs/cachefs/misc.c:271: warning: long long unsigned int format, different type arg (arg 2)
fs/cachefs/misc.c:271: warning: long long unsigned int format, different type arg (arg 3)

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/cachefs/misc.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -puN fs/cachefs/misc.c~cachefs-warning-fix-2 fs/cachefs/misc.c
--- 25/fs/cachefs/misc.c~cachefs-warning-fix-2	2004-09-08 01:54:18.292375472 -0700
+++ 25-akpm/fs/cachefs/misc.c	2004-09-08 01:54:43.157595384 -0700
@@ -266,8 +266,8 @@ void dump_bio(struct bio *bio, int n)
 
 	printk("BIO %d\n",n);
 	printk("\t- sector=%llu (bix=%llu) size=%x\n",
-	       bio->bi_sector,
-	       bio->bi_sector >> (PAGE_SHIFT - bits),
+	       (unsigned long long)bio->bi_sector,
+	       (unsigned long long)bio->bi_sector >> (PAGE_SHIFT - bits),
 	       bio->bi_size);
 	printk("\t- rw=%lx flags=%lx vcnt=%u/%u\n",
 	       bio->bi_rw,
_
